tests/sqlparser_snowflake.rs

```diff
@@ -2783,6 +2783,14 @@ fn test_parentheses_overflow() {
 }
 
 #[test]
+
+fn test_nested_select_with_lateral_flatten() {
+    let sql = "SELECT a FROM b, LATERAL FLATTEN(input => events)";
+    let _ = snowflake().parse_sql_statements(&sql).unwrap();
+
+    let sql = "SELECT (SELECT a FROM b, LATERAL FLATTEN(input => events))";
+    let _ = snowflake().parse_sql_statements(&sql).unwrap();
```

Comment:
hmm, the test function looks incomplete, missing a closing `}`? unless github is masking that line somehow :thinking:

Reply:
don't see anything unusual and test are passing, guess this is some github artifact.